doc/source/whatsnew/v0.20.0.txt

```diff
@@ -53,6 +53,9 @@ Other API Changes
 Deprecations
 ^^^^^^^^^^^^
 
+- ``Series.repeat()`` has deprecated the ``reps`` parameter in favor of ``repeats`` (:issue:`12662`)
+- ``Index.repeat()`` and ``MultiIndex.repeat()`` have deprecated the ``n`` parameter in favor of ``repeats`` (:issue:`12662`)
+- ``Categorical.searchsorted()`` and ``Series.searchsorted()`` have deprecated the ``v`` parameter in favor of ``key`` (:issue:`12662`)
```

Comment:
I don't see a reason to follow the `numpy` convention, especially since it is very unclear what exactly that parameter is.  Unless someone strongly objects, I would prefer to use `key`.